lmod_proxy/tests/test_edx_grades.py

```diff
@@ -254,10 +256,12 @@ def test_post_grades(self, mock_io):
         gradebook_return['data']['results'] = ['completely unexpected']
         gradebook.spreadsheet2gradebook.side_effect = None
 
-        with mock.patch(
-                'lmod_proxy.edx_grades.actions.render_template'
-        ) as mock_template:
-            message, data, success = post_grades(gradebook, form)
+        with self.app.app_context():
+            with mock.patch(
+                    'lmod_proxy.edx_grades.actions.render_template'
+            ) as mock_template:
+                message, data, success = post_grades(gradebook, form)
+
             self.assertTrue(gradebook.spreadsheet2gradebook.called)
             mock_io.assert_called_with(self.FULL_FORM['datafile'])
             mock_template.assert_called_with(
```

Comment:
Isn't this part of the block supposed to be indented more as well? It seems that `mock_template` is not in scope as-is.